GetandTake/DataAccess/Repositories/IEntityRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using GetandTake.Models.BaseModels;
+using System.Linq.Expressions;
+using System.Security.Principal;
+
+namespace GetandTake.DataAccess.Repositories
+{
+    public interface IEntityRepository<TEntity>
```

Comment:
You can easily take a look at my base CRUD repo [here](https://github.com/dotnet-fizzyy/Diploma/blob/563fdf10ed5310bb5dd685cbe833a026cc28fe55/WebAPI/WebAPI/Core/Interfaces/Database/IBaseCrudRepository.cs) and easily take whatever you wish too. I was trying to apply the most suitable and useful practices :)